### PR TITLE
Juniper SRX240 series device details updated. 

### PR DESCRIPTION
**Pull request by cieska**

### CODE REVIEW

**Comment by danner26** [comment]
Hello, thank you for contributing. Please resolve all failed test cases before marking this PR as ready to review. Thank you

**Comment by cieska** [comment]
> Hello, thank you for contributing. Please resolve all failed test cases before marking this PR as ready to review. Thank you

Hi.
I don't understand what's wrong. Why it didn't passed the checks. Can you lead me where the issue is, please?

**Comment by danner26** [comment]
So it looks like you made this on the master branch, and I can't update the branch for you. I think you might be behind on the latest version. I suggest you reset your fork, update to the latest version, then make a new branch off of master in your fork and make your changes. Then submit that as a PR.

I am sorry I cannot assist further at the moment.

**Comment by harryajc** [comment]
HI @cieska 
It looks like you are past the character limit on a single line for comments

File: device-types/Juniper/SRX240H2-POE.yaml [hunk 1]
```diff
@@ -8,7 +8,9 @@ front_image: true
 rear_image: true
 weight: 5.6
 weight_unit: kg
-comments: '[Juniper SRX240 Services Gateway Hardware Guide](https://www.juniper.net/documentation/en_US/release-independent/junos/information-products/pathway-pages/hardware/srx240/index.pdf)'
+airflow: right-to-left
+comments: '[Juniper SRX240 Services Gateway Hardware Guide - PDF](https://www.juniper.net/documentation/en_US/release-independent/junos/information-products/pathway-pages/hardware/srx240/index.pdf).
+  Highest supported JUNOS software release 12.3X48-D105 (04 Aug 2020) (only with 2GB RAM). [Upgrade SRX RAM guide](https://michaeldale.com.au/archive/2014/08/23/running-junos-121x47-on-first-gen-srx240h/).'
```
Comment (danner26) [inline device-types/Juniper/SRX240H2-POE.yaml:13]: I do not believe we should have links to unofficial websites within these files. All links should only be to manufacturer websites.

File: device-types/Juniper/SRX240H2.yaml [hunk 2]
```diff
@@ -7,7 +7,9 @@ front_image: true
 rear_image: true
 weight: 5.1
 weight_unit: kg
-comments: '[Juniper SRX240 Services Gateway Hardware Guide](https://www.juniper.net/documentation/en_US/release-independent/junos/information-products/pathway-pages/hardware/srx240/index.pdf)'
+airflow: right-to-left
+comments: '[Juniper SRX240 Services Gateway Hardware Guide - PDF](https://www.juniper.net/documentation/en_US/release-independent/junos/information-products/pathway-pages/hardware/srx240/index.pdf).
+  Highest supported JUNOS software release 12.3X48-D105 (04 Aug 2020) (only with 2GB RAM). [Upgrade SRX RAM guide](https://michaeldale.com.au/archive/2014/08/23/running-junos-121x47-on-first-gen-srx240h/).'
```
Comment (danner26) [inline device-types/Juniper/SRX240H2.yaml:12]: I do not believe we should have links to unofficial websites within these files. All links should only be to manufacturer websites.

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Hello @cieska as states on the individual files I do not think we should have links to external websites that are non-official documentation. Please remove these.

**Comment by harryajc** [comment]
As these unofficial links is all that has changed with this PR I don't think it is required the comments section already has a link to the Juniper websites documentation for these devices

**Pull request closed without merging**